packages/api/lambdas/process-s3-dead-letter-archive.js

```diff
@@ -87,13 +91,23 @@ async function processDeadLetterArchive({
   path = `${process.env.stackName}/dead-letter-archive/sqs/`,
   writeRecordsFunction = writeRecords,
   batchSize = 1000,
+  concurrency = 10,
```

Comment:
concurrency default is 30, it also set to different value 20 in another place. Should we make it consistent?
Same for dbMaxPool.

Reply:
👍🏻  cbdc9653f6b - good suggestion